backend/src/exercise/patient-ticking.ts

```diff
@@ -96,13 +73,14 @@ function getDedicatedResources(
             san: 0,
         };
     }
-    const material = Object.values(state.materials).filter((thisMaterial) =>
-        Object.keys(thisMaterial.assignedPatientIds).includes(patient.id)
-    ).length;
-    const treatingPersonnel = Object.values(state.personnel).filter(
-        (thisPersonnel) =>
-            Object.keys(thisPersonnel.assignedPatientIds).includes(patient.id)
+
+    const material = Object.keys(patient.assignedMaterialIds).length;
+
+    // TODO: check if this is efficient and maybe do it via for each assignedPersonnelId state.personnel[assignedPersonnelId]
```

Comment:
TODO: maybe save the currently treated patients in the personnel too for better performance.

Reply:
They are saved in the personnel, does the code look good to you now?